tests/scripts/test-e2e-standalone.sh

```diff
@@ -84,4 +84,10 @@ fi
 set -e
 
 echo Done waiting for OLS server start, running e2e
-SUITE_ID=standalone TEST_TAGS="not cluster and not rag" make test-e2e
+
+if [[ $MODEL = *"gpt"* ]]; then
+  EVAL_MODEL="gpt"
```

Comment:
rather than doing this here, i'd prefer the test code make this determination.

we have too much logic in our Makefile + test execution scripts already (for which i share the blame), it makes it hard for people to understand the flow as well as makes it harder to reuse the tests in new environments